### PR TITLE
Improvement in LabyrinthJS.activity toolbar

### PR DESCRIPTION
**Pull request by S-kus**
Fixes: #1073 
- Fixed overlapping of `sub-toolbar` with `activity-palette` by setting the z-index of palette higher than sub-toolbar.

### CODE REVIEW

File: activities/LabyrinthJS.activity/lib/sugar-web/activity/activity.js [hunk 1]
```diff
@@ -58,6 +58,8 @@ define(["webL10n",
         var activityPalette = new activitypalette.ActivityPalette(
             activityButton, datastoreObject);
 
+        document.getElementById("activity-palette").style.zIndex = "3";
```
Comment (llaske) [inline activities/LabyrinthJS.activity/lib/sugar-web/activity/activity.js:61]: You must not change code in sugar-web library. The code for this library must be the same for all activities.

Comment (S-kus) [reply activities/LabyrinthJS.activity/lib/sugar-web/activity/activity.js:61]: Okay sorry @llaske. 
I will try to solve this bug via styling changes in .css files.

**Comment by S-kus** [comment]
@llaske I fixed it by adding style in activity.css, could you please review it again.

**Comment by llaske** [comment]
Nice. Could remove change on activity.js from the PR?
Only the updated file should be in the PR.

**Comment by S-kus** [comment]
> Nice. Could remove change on activity.js from the PR? Only the updated file should be in the PR.

Yes sorry for that.
I corrected it now.

**Comment by llaske** [comment]
Nice. Thanks.

**Pull request merged**